### PR TITLE
Modifications

### PR DESCRIPTION
**Pull request by agibsonccc**
Updating examples; NOT MERGING YET.

### CODE REVIEW

File: src/main/java/org/deeplearning4j/ml/JavaMnistClassification.java [hunk 1]
```diff
@@ -41,16 +41,14 @@
     final static int outputNum = 10;
 
     public static void main(String[] args) {
-        SparkConf conf = new SparkConf()
+        SparkConf conf = new SparkConf().setMaster("local[*]")
```
Comment (EronWright) [inline src/main/java/org/deeplearning4j/ml/JavaMnistClassification.java]: Please don't merge this line.

Comment (agibsonccc) [inline src/main/java/org/deeplearning4j/ml/JavaMnistClassification.java]: I'm not. This is why it's a pr.

File: pom.xml [hunk 2]
```diff
@@ -56,37 +56,34 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_2.10</artifactId>
       <version>${spark.version}</version>
-      <scope>provided</scope>
```
Comment (EronWright) [inline pom.xml]: How are you running the example?   Are you attempting to run this in intellij?

Comment (agibsonccc) [inline pom.xml]: Yeah. Trying to understand this vs how I do things. Would like to compare
spark.ml vs our older spark

**Comment by EronWright** [comment]
Closing this to ensure it is not merged.    (shading, setMaster are a problem)

The log4j file is a good idea, I'll take care of that in another PR.

**Pull request closed without merging**